lib/utils/BaseReporter.js

```diff
@@ -173,6 +386,29 @@ class BaseReporter extends events.EventEmitter {
         return `\u001b[${COLORS[type]}m${str}\u001b[0m`
     }
 
+    limit (val) {
+        switch (Object.prototype.toString.call(val)) {
+        case '[object String]':
+            // Check the first 100 characters to detect a base64 string: http://stackoverflow.com/a/475217/67190
```

Comment:
I just merged #990. @sbmaxx used the [validator](https://www.npmjs.com/package/validator) package to detect base64, let's do that here too.